DependencyInjection/GosWebSocketExtension.php

```diff
@@ -30,6 +30,18 @@ public function load(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $configs = $this->processConfiguration($configuration, $configs);
 
+        // Set the SecurityContext for Symfony <2.6
+        // Should go back to simple configuration after drop <2.6 support
+        if (interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
```

Comment:
If we are sure about symfony version for this feature it's better to directly use the `VERSION_ID` provided by the kernel. What do you think ?

https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/HttpKernel/Kernel.php#L64